### PR TITLE
Fix 1179 - Add SUSI.AI for Magic Mirror, Yaydoc, Meilix and Phimp.me in Projects section under the menus icon in the navbar

### LINKED ISSUE #1179: Add SUSI.AI for Magic Mirror, Yaydoc, Meilix and Phimp.me in Projects section under the menus icon in the navbar
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

Presently we have only badgeyay,eventyay,loklak,susi,fossasia and pslab which appears on clicking on menus icon on the navbar. We should include Phimp.me, Meilix, Yaydoc and SUSI.AI for Magic Mirror also as they are important FOSSASIA projects. The meilix project already includes them.

**Screenshots of the issue:**

Meilix navbar:
aspectratioprb
SUSPER navbar:
contactinsusper

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1179 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:

- Fix #1179 

- https://pr-1215-fossasia-susper.surge.sh

### CODE REVIEW

**Comment by sk9331657** [comment]
@simsausaurabh  @praveenojha33  Can you please review this ??

**Comment by sk9331657** [comment]
@simsausaurabh  @fossasia/susper  @praveenojha33  @ParthS007  Can you please review ??

**Comment by sk9331657** [comment]
@praveenojha33  Fixed now.

**Comment by sk9331657** [comment]
@simsausaurabh  Please review .

**Comment by praveenojha33** [comment]
@sk9331657 The error in console is still present, also the link to code section is incorrect. Please fix both.

**Comment by sk9331657** [comment]
> @sk9331657 The error in console is still present, also the link to code section is incorrect. Please fix both.
Will do that .

**Comment by sk9331657** [comment]
> @sk9331657 The error in console is still present, also the link to code section is incorrect. Please fix both.

@praveenojha33  There are no console errors in pr link. 
@simsausaurabh  Please review.
screenshot from 2018-11-21 17-47-58

**Comment by sk9331657** [comment]
@simsaurabh Please review the pr.

File: src/app/dropdown/dropdown.component.css [hunk 1]
```diff
@@ -11,6 +11,16 @@ li.dropdown-menu-box {
     color: #737373;
     text-decoration: none;
 }
+::-webkit-scrollbar { 
+    display: none; 
+}
+
```
Comment (simsausaurabh) [inline src/app/dropdown/dropdown.component.css]: @sk9331657 Everything looks good, please remove the unwanted spaces. Keep the codebase clean.

**Comment by sk9331657** [comment]
@simsausaurabh  @praveenojha33  Please review.

**Comment by sk9331657** [comment]
@fossasia/loklak-devs @simsausaurabh   Please review.

**Comment by sk9331657** [comment]
@mariobehling  Please review and merge this

**Comment by sk9331657** [comment]
@praveenohja33 Can you please point out which link is not working bcoz I have tested all links and they are working??

**Comment by praveenojha33** [comment]
image
The code icon's link in service menu.

**Comment by sk9331657** [comment]
@praveenojha33  Please review now. Done required changes.

**Comment by sk9331657** [comment]
@simsausaurabh  @mariobehling  @fossasia/susper This can be now merged .

**Pull request merged**